vocabularies/voc4cat/0007228.ttl

```diff
@@ -5,7 +5,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007228>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007228"^^xsd:token ;
-    skos:definition "The number of cycles in a measurment, such as an cyclic voltammetry."@en ;
+    skos:definition "The number of cycles in a measurement, such as a cyclic voltammetry."@en ;
     skos:historyNote "0009-0008-1278-8890 Michael Götte created the resource"@en ;
     skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
     skos:prefLabel "number of cycles"@en ;
```

Comment:
Is this concept required if "voltametry cylcles" (#0007233) is defined?

Reply:
It is an important quantity in a cyclic voltammetry. Not sure why it shouldnt be needed. A cycle is one time from start to end and the number of cycles defines how often. for me these are two different things

Reply:
hey david, if so it would be an attribute of a cyclic voltammetry, but nevertheless atributes should have an IRI right?